eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/constants/EventMeshConstants.java

```diff
@@ -152,11 +152,15 @@ public class EventMeshConstants {
     public static final String MESSAGE = "message";
     public static final String CMD = "cmd";
     public static final String ACL = "acl";
+    public static final String BATCH_MSG = "batchMessage";
     public static final String MSG_TYPE = "msgtype";
     public static final String PERSISTENT = "persistent";
     public static final String HANDLER_ORIGIN = "Access-Control-Allow-Origin";
     public static final String HANDLER_METHODS = "Access-Control-Allow-Methods";
     public static final String HANDLER_HEADERS = "Access-Control-Allow-Headers";
     public static final String HANDLER_AGE = "Access-Control-Max-Age";
     public static final String MAX_AGE = "86400";
+
+    public static final String TCP_MONITOR = "tcpMonitor";
+    public static final String APP_MONITOR = "appMonitor";
 }
```

Comment:
How about combining TCP_MONITOR, APP_MONITOR, and BATCH_MSG together?

Reply:
Well, It will be better to move L164,165 to L155, because we have `message` and `acl` logger located at L153,154.

Reply:
Sure, makes sense, done as suggested